modules/acs-architecture-overview.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * architecture/index.adoc
+
+:_module-type: REFERENCE
+[id="acs-architecture_{context}"]
+= {product-title} architecture
+
+{product-title} installs as a pod in your OpenShift or a Kubernetes cluster and it includes the following components:
+
+|===
+| Component | Description | Quantity
+
+| <<central,*Central*>>
```

Comment:
Unfortunately we can't even do cross references like these in a module (even linking within the same module). Yet. So you'll have to remove these for now. But I like the TODOs you've added so that you can come back and add in these links later, once we have the xref implementation figured out and restriction removed. 

So I'd remove these for now, but add your TODO to come back and add them back later.